.github/stale.yml

```diff
@@ -1,24 +1,41 @@
 # https://github.com/marketplace/stale
+# https://github.com/probot/stale
+
+issues:
+    # Number of days of inactivity before an issue becomes stale
+    daysUntilStale: 30
+    # Number of days of inactivity before a stale issue is closed
+    daysUntilClose: 7
+    # Issues with these labels will never be considered stale
+    exemptLabels:
+      - Important
+      - Priority
```

Comment:
Small suggestion: Maybe we would also keep good first issue, always great start for new contributors.